poorwsgi/response.py

```diff
@@ -420,6 +406,41 @@ def __init__(self, arg: Union[int, Response], **kwargs):
         assert isinstance(arg, (int, Response))
         super().__init__(arg, kwargs)
 
+    def make_response(self):
+        """Return or make response if is possible."""
+        if isinstance(self.args[0], Response):
+            return self.args[0]
+
+        status_code = self.args[0]
+        if status_code == DECLINED:
+            return Decline()   # decline the connection
```

Comment:
Now finally a non-grammatical comment :) Why not include the meaning from the comment in the name of the class?
If you replace `Decline` with `DeclineConnection` you need no comment here.